src/ViewHoldingsRecord.js

```diff
@@ -269,6 +269,11 @@ class ViewHoldingsRecord extends React.Component {
     return false;
   };
 
+  getHoldingsNotes = (noteTypes, notes) => {
+    return notes
+      .filter((noteType) => noteTypes.find(note => note.holdingsNoteTypeId === noteType.id));
+  };
+
```

Comment:
There is an unnecessary return. If the arrow function has only return operator, you can remove it and curly braces.
```suggestion
getHoldingsNotes = (noteTypes, notes) => notes.filter((noteType) => noteTypes.find(note => note.holdingsNoteTypeId === noteType.id));
```

Reply:
Thanks, I'll fix it